mmbexplorer/src/MMBFile.cpp

```diff
@@ -3,34 +3,39 @@
 #include <algorithm>
 #include <string.h> // for memset
 #include "MMBFile.h"
-
-const unsigned char firstDirectoryEntry  [MMB_DIRECTORYENTRYSIZE] = { 0,1,2,3,0,0,0,0,0,0,0,0,0,0,0,0    };
-const unsigned char emptyDirectoryEntry  [MMB_DIRECTORYENTRYSIZE] = { 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,MMB_DISKATTRIBUTE_UNFORMATTED };
-const unsigned char invalidDirectoryEntry[MMB_DIRECTORYENTRYSIZE] = { 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,MMB_DISKATTRIBUTE_INVALID     };
+#include <io.h>
```

Comment:
Could this io.h include be removed? For compiling on MacOS and Linux, I need to remove it because it throws a file not found error. I haven't seen anything using it, is it Windows specific perhaps?

Reply:
this should have been windows specific - i have added the necessary changes to fix this.